src/materials/MazarsDamage.C

```diff
@@ -0,0 +1,145 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/*                       BlackBear                              */
+/*                                                              */
+/*           (c) 2017 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "MazarsDamage.h"
+
+#include "RankTwoTensor.h"
+#include "RankFourTensor.h"
+#include "ElasticityTensorTools.h"
+#include "MooseUtils.h"
+
+#include "libmesh/utility.h"
+
+registerMooseObject("BlackBearApp", MazarsDamage);
+
+template <>
+InputParameters
+validParams<MazarsDamage>()
+{
+  InputParameters params = validParams<DamageBase>();
+  params.addClassDescription("Mazars scalar damage model");
+  params.addParam<bool>(
+      "use_old_damage",
+      false,
+      "Whether to use the damage index from the previous step in the stress computation");
+  params.addRequiredParam<Real>("tensile_strength",
+                                "Tensile stress threshold for damage initiation");
+  params.addRequiredParam<Real>("a_t", "A_t parameter that controls shape of the response");
```

Comment:
The description should precise "in tension" for the ``_t`` parameters, and "in compression" for the ``_c`` parameters